cms/static/js/views/video_thumbnail.js

```diff
@@ -162,9 +204,16 @@ define(
             },
 
             imageSelected: function(event, data) {
-                this.readMessages([gettext('Video image upload started')]);
-                this.showUploadInProgressMessage();
-                data.submit();
+                var errorMessage = this.validateImageFile(data.files[0]);
```

Comment:
A miner bug just found: Need to remove error message after we select new image.
@muhammad-ammar @muzaffaryousaf FYI